src/fidesops/schemas/saas/saas_config.py

```diff
@@ -214,6 +222,10 @@ def get_graph(self) -> Dataset:
                         name=endpoint.name,
                         fields=fields,
                         grouped_inputs=grouped_inputs,
+                        after={
+                            CollectionAddress(*s.split("."))
+                            for s in endpoint.after or []
```

Comment:
`endpoint.after` will default to `[]` so we don't need `or []` unless you think it helps readability

Reply:
Ah I added this for mypy, I think if I remove the Optional from endpoint.after it should work.